internal/cypress/config.go

```diff
@@ -68,9 +69,13 @@ type Suite struct {
 
 // SuiteConfig represents the cypress config overrides.
 type SuiteConfig struct {
-	TestFiles         []string          `yaml:"testFiles,omitempty" json:"testFiles"`
+	TestFiles         []string          `yaml:"testFiles,omitempty" json:"testFiles,omitempty"`
 	ExcludedTestFiles []string          `yaml:"excludedTestFiles,omitempty" json:"ignoreTestFiles,omitempty"`
 	Env               map[string]string `yaml:"env,omitempty" json:"env"`
+
+	// introduced by cypress 10
+	SpecPattern        []string `yaml:"specPattern,omitempty" json:"specPattern,omitempty"`
+	ExcludeSpecPattern []string `yaml:"excludeSpecPattern,omitempty" json:"excludeSpecPattern,omitempty"`
```

Comment:
I don't think that's a good idea. Go has a pretty well established practice of versioning the entire package instead. Doing it the way this PR has it would introduce a lot of conditionals, making it even harder to remove further down the road. After all, the backwards compatible code is only needed for as long as we support those old versions.

Reply:
Oh I see!